uptane-standard.md

```diff
@@ -269,16 +269,16 @@ An OEM has issued a recall to address a problem with a keyless entry device that
 
 The OEM wants to use delta updates to save over-the-air bytes. The delta images contain only the code and/or data that has changed from the previous image version. To do so, the OEM must first modify the director repository, using the vehicle version manifest and dependency resolution to determine the differences between the previous and latest images. The OEM then adds the following to the custom targets metadata used by the director repository: (1) the algorithm used to apply a delta image, and (2) the targets metadata about the delta image. The OEM will also check whether the delta images match the targets metadata from the director repository.
 
-## Exceptions
-
-There are a number of factors that could impede the completion of the above scenarios:
-
-* ECUs may be lacking the necessary resources to function as designated. These resources could include weaknesses, in terms of CPU or RAM, that prevent performance of public key cryptography; a lack of sufficient storage to undo installation of bad software; or an ECU simply may reside on a low-speed network (e.g., LIN).
-* ECUs may reside on different network segments, and may not be able to directly reach each other, requiring a gateway to facilitate communication.
-* A user may replace OEM-installed ECUs with aftermarket ECUs.
-* A vehicle may be able to download only a limited amount of data via a cellular channel, due to limits on a data plan.
-* A system may lack sufficient power to download or install software updates.
-* Vehicles may be offline for extended periods of time, thus missing required updates (e.g., key rotations).
+## Exceptions
+
+There are a number of factors that could impede the completion of the above scenarios:
+
+* ECUs may be lacking the necessary resources to function as designated. These resources could include weaknesses, in terms of CPU or RAM, that prevent performance of public key cryptography; a lack of sufficient storage to undo installation of bad software; or an ECU simply may reside on a low-speed network (e.g., LIN).
+* ECUs may reside on different network segments, and may not be able to directly reach each other, requiring a gateway to facilitate communication.
+* A user may replace OEM-installed ECUs with aftermarket ECUs.
+* A vehicle may be able to download only a limited amount of data via a cellular channel, due to limits on a data plan.
+* A system may lack sufficient power to download or install software updates.
+* Vehicles may be offline for extended periods of time, thus missing required updates (e.g., key rotations).
```

Comment:
What are the exact changes here? It's hard to see what changed, if at all...

Reply:
Looks like someone stuck some DOS/Windows line endings (CRLF instead of LF) in there, and my text editor automatically fixed them. No content changes.